apps/next/app/api/deploy/up/route.ts

```diff
@@ -21,6 +21,9 @@ export async function POST(request: NextRequest) {
   const tempDir = 'temp';
   if (!existsSync(tempDir)) {
     mkdirSync(tempDir);
+  } else {
+    rmdirSync(tempDir, { recursive: true });
+    mkdirSync(tempDir);
```

Comment:
the tempdir needs to be created here in this endpoint and it needs to be different on every request

Reply:
Agreed. Please see this earlier response:
https://github.com/onmetal-dev/metal/pull/21#discussion_r1602132061